go.work.sum

```diff
@@ -2043,6 +2043,7 @@ golang.org/x/net v0.20.0/go.mod h1:z8BVo6PvndSri0LbOE3hAn0apkU+1YvI6E70E9jsnvY=
 golang.org/x/net v0.23.0/go.mod h1:JKghWKKOSdJwpW2GEx0Ja7fmaKnMsbu+MWVZTokSYmg=
 golang.org/x/net v0.25.0/go.mod h1:JkAGAh7GEvH74S6FOH42FLoXpXbE/aqXSrIQjXgsiwM=
 golang.org/x/net v0.26.0/go.mod h1:5YKkiSynbBIh3p6iOc/vibscux0x38BZDkn8sCUPxHE=
+golang.org/x/net v0.28.0/go.mod h1:yqtgsTWOOnlGLG9GFRrK3++bGOUEkNBoHZc8MEDWPNg=
```

Comment:
It's unclear to me why this was not committed?  It might be worth a CI check to ensure the go.* is not mutated on build.

Reply:
We probably should add this file to .gitignore. This file is very noisy, as it is dependent on the union of all transitive deps for the repository and the patch version of go used to generate it.

This and the .work file aren't used downstream. The release CI deletes this file and recreates it as needed. Both are checked in as a convenience for developers. Previously, we had developers run a shell script to initialize the workspace, but that proved too difficult for people onboarding.